content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -21,7 +21,7 @@ further_reading:
 
 Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files.
 
-The .NET Tracer uses the [LibLog][2] library to automatically inject trace IDs, span IDs, `env`, `service`, and `version` into your application logs. If you haven't done so already, it is recommended to configure the .NET tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the best experience when adding `env`, `service`, and `version` (see [Unified Service Tagging][3] for more details).
```

Comment:
Did you remove "into your application logs" at the end of the sentence intentionally? I don't think the sentence makes much sense without that.

Reply:
Good catch!

Reply:
Updated!